e2e/pvc.go

```diff
@@ -74,6 +74,73 @@ func createPVCAndvalidatePV(c kubernetes.Interface, pvc *v1.PersistentVolumeClai
 	})
 }
 
+func createPVCAndPV(c kubernetes.Interface, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume) error {
+	_, err := c.CoreV1().PersistentVolumeClaims(pvc.Namespace).Create(context.TODO(), pvc, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	_, err = c.CoreV1().PersistentVolumes().Create(context.TODO(), pv, metav1.CreateOptions{})
```

Comment:
if this fails, the PVC was created already. Maybe log that it leaks, or delete it?

Reply:
currently, if something fails we are not deleting/cleaning up the resources. the error will be more evident from the Create function will take care of cleanup in followup patches we need to change few big things in the E2E framework